lib/msal-common/src/authority/Authority.ts

```diff
@@ -433,9 +435,25 @@ export class Authority {
         return matches.length > 0;
     }
 
+    /**
+     * helper function to populate the authority based on azureCloudOptions
+     * @param authorityString
+     * @param azureCloudOptions
+     */
+    static generateAuthority(authorityString: string, azureCloudOptions?: AzureCloudOptions): string {
+        let authorityAzureCloudInstance;
+
+        if (azureCloudOptions && azureCloudOptions.azureCloudInstance !== AzureCloudInstance.None) {
```

Comment:
What if `azureCloudOptions.azureCloudInstance` is not defined?

Reply:
It defaults to `None`, so this should be fine.

Reply:
It's an optional parameter. You should either make it required if it always has a value or check that it's defined when you reference it.

Reply:
Guess I can make `azureCloudInstance` mandated and `azureCloudOptions` optional. Will be consistent. Thanks. Good catch.

Reply:
Done